ui/src/app/machines/components/ActionFormWrapper/CloneForm/CloneFormFields/SourceMachineSelect/SourceMachineDetails/SourceMachineDetails.tsx

```diff
@@ -0,0 +1,98 @@
+import pluralize from "pluralize";
+
+import LabelledList from "app/base/components/LabelledList";
+import Placeholder from "app/base/components/Placeholder";
+import type { MachineDetails } from "app/store/machine/types";
+
+type Props = {
+  machine: MachineDetails | null;
+};
+
+export const SourceMachineDetails = ({ machine }: Props): JSX.Element => {
+  if (!machine) {
+    return (
+      <LabelledList
+        data-test="placeholder-list"
+        items={[
+          { label: "Status", value: <Placeholder>Deployed</Placeholder> },
+          {
+            label: "CPU",
+            value: (
+              <>
+                <Placeholder>X cores, X.X GHz</Placeholder>
+                <br />
+                <Placeholder>Vendor information</Placeholder>
+                <br />
+                <Placeholder>Architecture</Placeholder>
+              </>
+            ),
+          },
+          { label: "Memory", value: <Placeholder>X GiB</Placeholder> },
+          {
+            label: "Storage",
+            value: <Placeholder>X GB over X disks</Placeholder>,
+          },
+          { label: "Power type", value: <Placeholder>Power type</Placeholder> },
+          { label: "Owner", value: <Placeholder>Owner</Placeholder> },
+          { label: "Host", value: <Placeholder>Host name</Placeholder> },
+          { label: "Zone", value: <Placeholder>Zone name</Placeholder> },
+          { label: "Domain", value: <Placeholder>Domain</Placeholder> },
+        ]}
+      />
+    );
+  }
+  const {
+    architecture,
+    cpu_count,
+    cpu_speed,
+    domain,
+    metadata,
+    memory,
+    owner,
+    physical_disk_count,
+    pod,
+    power_type,
+    status,
+    storage,
+    zone,
+  } = machine;
+  return (
+    <LabelledList
+      data-test="source-machine-details"
+      items={[
+        { label: "Status", value: status },
```

Comment:
I wonder if instead of duplicating this table we could update `Placeholder` to take the content and a loading prop so you could just do `<Placeholder loading={!machine} content="Deployed">{status}</Placeholder>` and then when it has loaded it `Placeholder` just returns the children.

What do you think?

Reply:
Yeah that sounds pretty nice! I'll create an issue for it.

https://github.com/canonical-web-and-design/maas-ui/issues/2933